script/6_deployUSDTContract.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "Setting environment variables..."
+source .env
```

Comment:
It looks like you want to run this script from the project root. Other deployment scripts are rather meant to be run from the script folder (and have an extra `cd ../` command at the beginning). I think we should be consistent

Reply:
Would instead `source ../.env` make sense ?
if we do `cd../` to go to root then would need to come back to script dir again.

Reply:
I have no preference, but I would be consistent across all scripts

Reply:
I updated per your suggestion

Reply:
maybe then we should also add
```
echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
set -e
echo "Done."
```

Reply:
just for consistency. it's one command so it doesn't change anything